Source/WebKit/WebProcess/GPU/media/RemoteAudioDestinationProxy.cpp

```diff
@@ -68,6 +68,9 @@ RemoteAudioDestinationProxy::RemoteAudioDestinationProxy(AudioIOCallback& callba
     , m_numberOfInputChannels(numberOfInputChannels)
     , m_remoteSampleRate(hardwareSampleRate())
 {
+    RunLoop::current().dispatch([protectedThis = Ref { *this }]() {
+        protectedThis->connection();
+    });
```

Comment:
Have we tested this change on all platforms? It seems [259134@main](https://commits.webkit.org/259134@main) and [261361@main](https://commits.webkit.org/261361@main) indicate not prewarming connection to GPU is good for iOS.

Reply:
That's a good point! I'll make sure to also test this on iOS.

Thanks for reviewing!